lmod_proxy/edx_grades/actions.py

```diff
@@ -30,6 +30,16 @@ def post_grades(gradebook, form):
     csv_file.seek(0)
     results = None
     try:
+        # Create assignment to explicitly set max points.
```

Comment:
Oh, there's one assignment per column, and one email per row. However the number of max_points isn't necessarily same for each row, at least in the example @pdpinch gave me